proxy/http/Http1ClientSession.cc

```diff
@@ -40,10 +40,10 @@
 
 #define HttpSsnDebug(fmt, ...) SsnDebug(this, "http_cs", fmt, __VA_ARGS__)
 
-#define STATE_ENTER(state_name, event, vio)                                                             \
-  do {                                                                                                  \
-    /*ink_assert (magic == HTTP_SM_MAGIC_ALIVE);  REMEMBER (event, NULL, reentrancy_count); */          \
-    HttpSsnDebug("[%" PRId64 "] [%s, %s]", con_id, #state_name, HttpDebugNames::get_event_name(event)); \
+#define STATE_ENTER(state_name, event, vio)                                                                      \
+  do {                                                                                                           \
+    /*ink_assert (magic == HTTP_SM_MAGIC_ALIVE);  REMEMBER (event, NULL, reentrancy_count); */                   \
+    HttpSsnDebug("[%" PRId64 "] [%s, %s]", connection_id(), #state_name, HttpDebugNames::get_event_name(event)); \
   } while (0)
```

Comment:
A way to avoid the long macro is to make
```
struct Dbg;
```
a private member of Http1ClientSession, and here put:
```
struct Http1ClientSession::Dbg
{
  static void
  state_enter(Http1ClientSession *this_, char const *state_name, int event)
  {
    /*ink_assert (magic == HTTP_SM_MAGIC_ALIVE);  REMEMBER (event, NULL, reentrancy_count); */
    HttpSsnDebug("[%" PRId64 "] [%s, %s]", this_->connection_id(), state_name, HttpDebugNames::get_event_name(event));
  }
};
#define STATE_ENTER(STATE_NAME, EVENT) Dbg::state_enter(this, #STATE_NAME, (EVENT))
```